hw3/01-characters.js

```diff
@@ -1,4 +1,73 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const getApi = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+/**
+ * Asynchronous function to fetch data from a specified URL
+ * @param {string} url - The URL from which to fetch data.
+ * @returns {Promise} - A promise that resolves with the fetched JSON data, or rejects with an error
+ */
+const fetchData = async (url) => {
+    let promise = new Promise((resolve, reject) => {
```

Comment:
Minor - Naming : Promise is named 'promise'. Suggestions : 'getCharacters' , 'fetchCharacters' etc.. 
Better would be to return promise directly!

Reply:
Replaced promise with getCharacters.